src/shared/tpm2-util.c

```diff
@@ -5891,10 +5891,7 @@ int tpm2_unseal_data(
                                        "Failed to unseal data: %s", sym_Tss2_RC_Decode(rc));
 
         _cleanup_(iovec_done) struct iovec d = {};
-        d = (struct iovec) {
-                .iov_base = memdup(unsealed->buffer, unsealed->size),
-                .iov_len = unsealed->size,
-        };
+        d = IOVEC_MAKE(memdup(unsealed->buffer, unsealed->size), unsealed->size);
```

Comment:
Not you fault, but missing oom check.

Reply:
There is an oom check just after the following `explicit_bzero_safe()`.

Reply:
Indeed. Sorry for the noise.